docs/en/revocation-lists.rst

```diff
@@ -1,57 +1,206 @@
+
+The "Non-Revocation Attestation Renewal Flow" and "Credential Revocation Flow" subsections provide detailed workflows for renewing a non-revocation attestation and revoking a credential, respectively. These workflows include the generation and verification of a Proof of Possession (PoP) JWT, which proves that the Wallet Instance possesses the private key associated with the VC.
+
+
+Key Technical Requirements
+---------------------------
+
+- The Authentic Source MUST be the only source of truth for credential validity. All revocation requests MUST be communicated to the Authentic Source using appropriate channels. These requests MAY come from:
+    - The citizen (Holder) via their personal Wallet Instance.
+    - The citizen via a legal entity (e.g., Police for reporting).
+    - The citizen directly via the Authentic Source.
+    - The Authentic Source itself (e.g., for attribute updates).
+    - A legal entity (e.g., Police for seizure).
+- The Authentic Source MUST maintain a record of the issuers who have issued a certain credential. 
+- The Issuer MUST provide information about a callback hook for revocation.
+- The Authentic Source MUST manage the revocation of issued credentials by notifying the respective issuers.
+- The Authentic Source MUST maintain a mapping between ID_CREDENTIAL and ID_ISSUER to facilitate credential revocation even in case of Wallet Instance loss.
+- The revocation MUST be verifiable via a cryptographic signature.
+
+
+Presentation
 ------------
 
- - req 1
- - req 2
+The Wallet Instance responds to a presentation request from the Relying Party. It produces a Verifiable Presentation (VP) token with the digital credential and a non-revocation attestation. This attestation is a proof that the credential has not been revoked.
+
+It is timestamped with the request date and always refers to a previous period. Importantly, this attestation does not reveal any information about the Relying Party. The Relying Party then evaluates and validates the response, the VP token, and the non-revocation attestation.
+
+Issuance
+--------
+
+The Wallet Instance initiates a request for a digital credential from the Credential Issuer.
+
+Upon successful request, the Wallet Instance receives:
+- The digital credential
+- A non-revocation attestation
+
+The non-revocation attestation is presented in conjunction with the credential. The attestation is timestamped with the request date, not the authentication date, and always refers to a previous period.
+Relying Parties determine the validity duration of this attestation based on the specific use case.
+By default, an attestation is considered valid if issued within the previous 350 hours. This provision supports offline use cases and potential disruptions or incidents in the certification systems. Note that this delay only refers to the non-revocation of the credential, while the validity of the credential itself can always be verified.
```

Comment:
It's to avoid potentially disastrous consequences due to unforeseen downtime: if non-revocation attestations can't be refreshed, we still want to have a grace period.